docs/src/content/docs/commands/SLEEP.md

```diff
@@ -53,18 +56,10 @@ Pause the client for 5 seconds.
 ```shell
 127.0.0.1:7379> SLEEP 5
 OK
+(5.01s)
```

Comment:
Is this required?

Reply:
I agree, does not look the best but I just copied the exact output so that it also demonstrates that `OK` is returned after 5 seconds have elapsed (as mentioned in the return values table) 
Should I remove it?

Reply:
Yes, I think seconds elapsed is not required.

Reply:
okay, thanks!